openshift/ftest-openshift-resources-standalone/src/test/java/org/arquillian/cube/openshift/standalone/HelloWorldOpenShiftResourcesTest.java

```diff
@@ -35,4 +41,12 @@ public void should_show_hello_world() throws IOException {
         assertThat(response.code()).isEqualTo(200);
         assertThat(response.body().string()).isEqualTo("Hello OpenShift!\n");
     }
+
+    @Test
+    @OpenShiftResource("classpath:hello-route-2.yaml")
+    public void should_register_extra_route() {
+        final RouteList routes = openShiftClient.routes().list();
+        assertThat(routes.getItems()).hasSize(2);
```

Comment:
It would be nice if we test some metadata about the route just created, instead of testing the number of routes. This way this test wouldn't break if we add other routes in the future (in the class level).